index.html

```diff
@@ -29,84 +29,68 @@
-          <p>Ampersand.js is a well-defined approach to combining (get it?) a series of intentionally tiny modules.</p>
-          <p>Read <a href="http://blog.andyet.com/2014/06/25/introducing-ampersand-js">the post introducing ampersand.js</a> on our blog for more background.</p>
-          <h3>Maximize simplicity</h3>
-          <p><em>If you don’t actively fight for simplicity in software, complexity will win, and it will suck.</em></p>
-          <ul>
-            <li>Everything is a CommonJS module.</li>
-            <li>Everything is installed via npm.</li>
-            <li>Everything is MIT licensed.</li>
-            <li>Trusted versioning with semver.</li>
-          </ul>
-          <h3>Designed for rich “app” experiences</h3>
-          <ul>
-            <li>Modern browsers by default. Unapologetically supporting only IE9+.</li>
-            <li>The browser is a runtime, not a document viewer.</li>
-            <li><3s realtime apps</li>
-          </ul><a href="/learn" class="button button-primary">Discover Ampersand.js</a>
-        </div>
-        <div class="releases-list">
-          <h3>Recent releases</h3>
-          <div class="module-card">
-            <h4 class="module-name">ampersand</h4>
-            <p class="module-version">v3.0.1 – Apr 21, 2015</p>
-            <p class="module-links"><a href="https://github.com/ampersandjs/ampersand">github</a><a href="https://www.npmjs.org/package/ampersand">npm</a></p>
-          </div>
-          <div class="module-card">
-            <h4 class="module-name">ampersand-router</h4>
-            <p class="module-version">v3.0.2 – Apr 08, 2015</p>
-            <p class="module-links"><a href="https://github.com/ampersandjs/ampersand-router">github</a><a href="https://www.npmjs.org/package/ampersand-router">npm</a></p>
-          </div>
-          <div class="module-card">
-            <h4 class="module-name">ampersand-select-view</h4>
-            <p class="module-version">v3.0.1 – Apr 08, 2015</p>
-            <p class="module-links"><a href="https://github.com/AmpersandJS/ampersand-select-view">github</a><a href="https://www.npmjs.org/package/ampersand-select-view">npm</a></p>
-          </div><a href="/contribute" class="button button-primary">How to contribute</a>
-        </div>
+        <h2>What is Ampersand.js?</h2>
+        <p>Imagine a version of <a href="http://backbonejs.org/">Backbone.js</a> where each piece (model, view, collection, etc) is its own <a href="http://github.com/ampersandjs">GitHub repository</a> and <a href="https://www.npmjs.com/search?q=ampersand">npm module</a> with good, unified <a href="/docs">documentation</a> (and yes, npm is <em>wonderful</em> for frontend code).</p>
+        <p><em>But</em>, it's not just that. We added features to some of those pieces and <a href="http://ampersandjs.com/docs#ampersand-state">replaced others entirely</a> but if you know Backbone.js you'll feel at home.</p>
```

Comment:
What features? This sentence sounds a little bit too vague to me. Does that mean you removed Backbone features and replaced them with something else? Needs to be more specific imho.